packages/core/test/event.test.js

```diff
@@ -1,12 +1,13 @@
-const { describe, it, expect } = global
```

Comment:
Only this file required actual adjustments to the test setup. The rest are lint changes mostly converting async tests to the promise approach.